lib/document/paged-rendering.js

```diff
@@ -0,0 +1,33 @@
+window.AsciidoctorPDF = window.AsciidoctorPDF || {}
+window.AsciidoctorPDF.status = 'rendering'
+class PagedReadyHandler extends Paged.Handler {
+  afterRendered(pages) {
+    window.AsciidoctorPDF.status = 'ready'
+  }
+}
+Paged.registerHandlers(PagedReadyHandler);
+
+window.PagedConfig = window.PagedConfig || {}
+
+window.PagedConfig.auto = false
+
+// same logic as pagedjs, but waiting for 'complete' instead of 'interactive'
+let ready = new Promise(function(resolve, reject){
+  if (document.readyState === "complete") {
+    resolve(document.readyState);
```

Comment:
I'm surprised, the linter should not be happy about this semi-colon... let me check.

Reply:
I will fix it in another pull request